app/models/person.rb

```diff
@@ -2,7 +2,6 @@
 #
```

Comment:
Would be nice to have a comment describing difference between Person and User...